surrealdb/__init__.py

```diff
@@ -1,5 +1,5 @@
 """
-Copyright © SurrealDB Ltd
+Copyright © SurrealDB Ltd.
```

Comment:
Why are there commas here?
```suggestion
Copyright © SurrealDB Ltd
```
Imo this looks better

Reply:
they aren't commas, they are periods

Reply:
> they aren't commas, they are periods

Thats what I meant, wasn't paying attention

Reply:
ltd is an abbreviation, so a period on the end is actually more correct. i think this is fine

Reply:
The period is there as one of the flake8 docstring rules is the first line of a docstring should end with a period, but as @tsunyoku said, ltd is an abbreviation and a period is correct in that case